### PR TITLE
Prettify

### LINKED ISSUE #15: UI.prettify()
We don't need the prettiest site in existence, but some extra UI work wouldn't hurt.

- Clean up base template
- `Boostrap.prettify()`
  + [ ] include the CSS in the base
  + [ ] add to existing templates

### LINKED ISSUE #25: Add cancel buttons to forms
Add cancel buttons that send you back to the previous page without saving changes

### PR DESCRIPTION
**Pull request by dealie16**
Added in bootstrap and styled a lot of the pages with it. Also added in Cancel buttons to some of the forms. resolves #15 and resolves #25

### CODE REVIEW

File: corvid/main/models/page.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 class Page(models.Model):
     title = models.CharField(max_length=50)
-    content = models.CharField(max_length=50000)
+    content = models.TextField()
```
Comment (jpcjr) [inline corvid/main/models/page.py]: I put these limits in for sanity's sake. I know Postgres doesn't care about string length under the hood, but hard drives do. And 50k is huge: 5x what Reddit allows per comment. Finally, we'll need to specify a widget in our forms anyway if we want to use our fancy editor for pages / posts, so using a TextField fixes only a temporary problem for us.

With that in mind, does this change still benefit us?

**Comment by jpcjr** [comment]
Pushed a change to unbork Edit Page. After we resolve the field change, this can be merged in. :+1:

File: corvid/main/models/page.py (hunk 1 above)
Comment (dealie16) [inline corvid/main/models/page.py]: I guess the only reason that I still like it being a TextField is that it enforces my mental model ie that content field accepts a long string and a TextField accepts a lot long string. I can also set max length in the TextField.

Comment (jpcjr) [inline corvid/main/models/page.py]: Cleaner mental model is valid. Does `TextField(max_length=50000)` in here (`Page`) and in `Post` work for you?

**Comment by brenns10** [comment]
M-m-m-m-merge!!

**Pull request merged**